src/StackAdmin/Azs.Subscriptions/Module/Azs.Subscriptions/Azs.Subscriptions.psd1

```diff
@@ -103,19 +103,27 @@ Licensed under the MIT License. See License.txt in the project root for license
         PSData = @{
 
             # Tags applied to this module. These help with module discovery in online galleries.
-            # Tags = @()
+            Tags         = @('AzureStack', 'Azure Stack', 'Tenant', 'Subscriptions', 'Azure Stack Tenant Subscription')
 
             # A URL to the license for this module.
-            # LicenseUri = ''
+            LicenseUri   = 'https://aka.ms/azps-license'
 
             # A URL to the main website for this project.
-            # ProjectUri = ''
+            ProjectUri   = 'https://github.com/Azure/azure-powershell'
 
             # A URL to an icon representing this module.
             # IconUri = ''
 
             # ReleaseNotes of this module
-            # ReleaseNotes = ''
+            ReleaseNotes = '## 2018.8.12 - Version 0.2.0
+            * Module dependencies updated
+                * AzureRM.Profile >= 5.3.4
+                * AzureRM.Resources >= 6.0.2
```

Comment:
even here please dont mention >= 6.0.2 lets just say greater than equal to dependency on the azurerm.resources module

The uber all release notes alone can mention the version

Reply:
Resolved